llvd/__init__.py

```diff
@@ -20,47 +24,44 @@
 
 def create_browser():
     threadLocal = threading.local()
-    browser = getattr(threadLocal, 'browser', None)
+    browser = getattr(threadLocal, "browser", None)
     if browser is None:
         options = webdriver.ChromeOptions()
         options.add_argument("--headless")
         options.add_argument("--disable-notifications")
         options.add_argument("--log-level=3")
-        options.add_argument('--ignore-certificate-errors')
-        options.add_argument('--ignore-ssl-errors')
+        options.add_argument("--ignore-certificate-errors")
+        options.add_argument("--ignore-ssl-errors")
         browser = webdriver.Chrome(options=options, service_log_path=None)
         browser.delete_all_cookies()
-        setattr(threadLocal, 'browser', browser)
+        setattr(threadLocal, "browser", browser)
         return browser
 
 
-@ click.command()
-@ click.option('--course', '-c', help='llvd --help')
+@click.command()
+@click.option("--course", "-c", help="llvd --help")
 def main(course):
     """
-    Linkedin learning video downloader cli tool\n
-    example: llvd --course "Java 8 Essential"\n
+    Linkedin learning video downloader cli tool
+    example: llvd --course "Java 8 Essential"
     Remember to set login credentials in the config file
     """
-    if(len(sys.argv) == 1):
-        print("missing required arguments: run llvd --help")
+    if len(sys.argv) == 1:
+        click.echo(f"{BOLD}Missing required arguments: llvd --help", err=True)
         sys.exit(0)
 
-    course = re.sub('[)|(|,]|(-&)', '', course.lower())
+    course = re.sub("[)|(|,]|(-&)", "", course.lower())
 
-    link = config.main_url + str(course).replace(" ", "-").replace(":-",
-                                                                   "-").replace("-&", "").replace(".", "-")
-    if(config.email == "" or config.password == ""):
-        print("missing credentials llvd --help")
-        sys.exit(0)
+    link = config.main_url + str(course).replace(" ", "-").replace(":-", "-").replace(
+        "-&", ""
+    ).replace(".", "-")
+    email = click.prompt("Please enter your linkedin email address: ")
+    password = click.prompt("Enter your Linkedin Password: ", hide_input=True)
     try:
         browser = create_browser()
     except WebDriverException:
-        print("chromedriver is missing")
+        click.echo(f"{BOLD}Please install chromedriver", err=True)
```

Comment:
The argument err=True throws a TypeError..you might want to remove it.

Reply:
That's very weird, [from click 3.0 to above, that error is not supposed to appear](https://click.palletsprojects.com/en/7.x/utils/).
Anyway, let me remove it.